spec/models/organization_spec.rb

```diff
@@ -262,4 +262,22 @@
       it { expect(organization.display_name).to be_nil }
     end
   end
+
+  describe '#immersible?' do
+    context 'current public' do
+      let(:one) { build(:organization, public: true) }
+
+      let(:other_immersive) { build(:organization, immersive: true) }
+      let(:other_non_immersive) { build(:organization, immersive: true) }
```

Comment:
I guess this one is meant to be false :thinking:, right? And the test below also has the wrong value